csrc/python_frontend/python_bindings.cpp

```diff
@@ -768,6 +768,28 @@ void initNvFuserPythonBindings(PyObject* module) {
   NVFUSER_PYTHON_BINDING_UNARY_OP("imag", imag)
 #undef NVFUSER_PYTHON_BINDING_UNARY_OP
 
+  // overload to
+  nvf_ops.def(
+      "set",
```

Comment:
Perhaps we should change this to `set_stride_order`.

Reply:
How about making  the `stride_order` argument keyword-only?

Reply:
Thunder has an equivalent function called `stride_order` and numpy has [numpy.lib.stride_tricks.as_strided](https://numpy.org/doc/stable/reference/generated/numpy.lib.stride_tricks.as_strided.html)

@jacobhinkle Did you mean to keep the `set` name but make `stride_order` a keyword argument?

Reply:
renamed it to stride_order now.

Reply:
soft cc'ing Jacob.

I made the signature as `stride_order(TensorView, list[int])`. so stride order is required now for the API.